photon-lib/src/main/java/org/photonvision/SimVisionTarget.java

```diff
@@ -25,6 +26,17 @@
     double targetHeightAboveGroundMeters;
     double tgtAreaMeters2;
 
+    /**
+    * Different pre-defined vision targets, used in constructor to create a target in its official
+    * position from center of feild.
+    */
+    public enum VisionTarget {
```

Comment:
Remove the switch below, and replace with something like the following:
```
public enum VisionTarget {
k2020RedPowerPort(new Pose2d(), 2, 0.99, 0.4....),
k2020BluePowerPort(new <foobar>);

public final Pose2d pose;
VisionTarget(pose2d, double, etc);
}
```

Something like this https://github.com/wpilibsuite/allwpilib/blob/main/wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/DifferentialDrivetrainSim.java#L389

Reply:
Also maybe consider refactoring this enum into its own file?